### ISSUE TITLE
Incorrect heading level for spotlight section on health services page

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#12080]
## Describe the defect
On VAMC System health services pages, there is a heading for the spotlight section that is currently an H3 but should be an H2 to properly fit the heading structure of the page.

## AC / Expected behavior

- [ ] Update "In the spotlight at [VAMC system name]" heading to be an H2
- [ ] Accessibility review 

## Screenshots
![image](https://user-images.githubusercontent.com/106678594/209156166-38339b17-2b48-486a-96a8-f8f8cc6f66f1.png)

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [x] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by briandeconinck** [comment]
This issue was identified during the Collaboration Cycle Staging Review for Lovell VAMC on 2/28/2023. Applicable Collaboration Cycle labels have been added to this ticket for tracking and dashboard purposes, rather than creating a duplicate ticket.

**Comment by briandeconinck** [comment]
Ticket referenced in [va.gov-team#64420](https://github.com/department-of-veterans-affairs/va.gov-team/issues/64420), which should be closed when this issue is closed.

**Comment by CherylEvans** [comment]
This ticket was recreated in va.gov-team ticket # [64838](https://app.zenhub.com/workspaces/collaboration-cycle-reviews-61d89e9e5f4c120011a0ad37/issues/gh/department-of-veterans-affairs/va.gov-team/64838)

**Comment by laflannery** [comment]
This is going to get resolved with the [Health services component update](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17657) which is in progress so I'm going to close this
